buf/registry/plugin/v1beta1/commit_service.proto

```diff
@@ -53,7 +53,7 @@ message GetCommitsRequest {
   // If this DigestType is not available, an error is returned.
   // Note that certain DigestTypes may be deprecated over time.
   //
-  // If not set, the latest DigestType is used, currently p1.
+  // If not set, the latest DigestType is used, currently P1.
```

Comment:
Is this supposed to be O1 too?

Reply:
No this is in `plugin`, plugin digest type is still `p1`.